README.md

```diff
@@ -46,7 +46,17 @@ check for properties order:
 }
 ```
 
-The `order` config enforces properties order given by following categories:
+The `order` config enforces a consistent order of content in your declaration blocks.:
+
+1. Sass variables,
+2. CSS custom properties,
```

Comment:
https://github.com/lmc-eu/spirit-design-system/blob/e3c385acf3e40e5318a39b4bf6dc3264cd67796b/packages/web/src/scss/components/Modal/_ModalDialog.scss#L20

Look at the ceiling. 👀

Reply:
OK, let's fix it there then.